src/composable/composable.js

```diff
@@ -0,0 +1,38 @@
+// https://stackoverflow.com/questions/72209080/vue-3-is-getcurrentinstance-deprecated
+import { getCurrentInstance } from "vue";
+import { harnessStore } from "../harness";
+
+export default function harnessComposable() {
```

Comment:
The docs say that the convention is to name composables `useX`, so it should be named something like `useHarness`

Reply:
Good point, and something I've confused myself with in nomenclature too. I've made that change across the codebase, ran tests, and loaded the plugin in my testing environment. This is a somewhat breaking change if anyone has a project utilizing the `harnessStore` in their code - but to my knowledge, we've so far just used the new harness-vue in my PSA project and @michael-long88's school project which likely doesn't dig that deep into it yet either. I can release as `1.2.0`

Reply:
I don't think the store would use the `use` nomenclature since it isn't a composable, that's the react hook standard and composables seem to be directly referencing hooks. And in that vein `useHarnessComposable` would be redundant since composable is directly implied by using `use`, but I'm not sure of correct terminology more specific than `useHarness`... `useHarnessPage`?

Reply:
The pinia doc examples tend to be `useXStore` formatted: https://pinia.vuejs.org/introduction.html#basic-example

Reply:
gotcha 👍